kafka-connect-upload-source/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ project.extra.apply {
     set("jacksonDataVersion", "2.9.10")
     set("openCsvVersion", "4.6")
     set("confluentVersion", "5.3.0")
-    set("radarSchemaVersion", "0.5.7")
+    set("radarSchemaVersion", "0.5.8-SNAPSHOT")
```

Comment:
Will you update this to stable version on release?

Reply:
yes. Planning to